woocommerce/woocommerce-android
# Add subscription sales price

Closes: #8754

### Description
Creating the subscription card for product details/variation details, I missed the sale information available to merchants before the subscription support. This PR fixes that by adding the sale information to the subscription card/detail screen.

pdcxQM-1YN-p2#comment-1860

> We’re not showing sale price in product details like we do for non-subscriptions. You can see the sale price from the product list (same as other types of orders), but the Subscription item within the order only shows the regular price.

### Testing instructions
TC1
1. On the `web-admin`, set a sale price for a simple subscription product
2. Open the subscription product using the app
3. Check that the subscription information matches with `web-admin` (sale price, sale dates)

TC2
1. On the `web-admin`, set a sale price for a variation on a variable subscription product
2. Open the subscription variation product using the app
3. Check that the subscription information matches with `web-admin` (sale price, sale dates)

### Images/gif

https://user-images.githubusercontent.com/18119390/230239494-1b0b8708-4218-407f-80c1-78034c38b567.mp4
